### DISCUSSION TITLE
Running test

### DISCUSSION THREAD

**Discussion by schriftgestalt** [fonttools/fonttools#2783] [Q&A] [answered]
I can’t figure out how to run the tests. Can someone please help me with that? I’m mostly interested in the ttLib test. It might be nice if the Tests folder had a Readme.

**Comment by anthrotype** [comment, marked as answer]
https://fonttools.readthedocs.io/en/latest/developer.html#testing

**Reply by anthrotype** [reply]
the "developer documentation" is linked to from the main README page

**Reply by anthrotype** [reply]
We can't explain in our own docs how to run pytest or tox, those have their own docs. pytest itself is basically the standard way of testing python. If you're interested in running a specific test, you would give `pytest` command the path to a specific test directory, test module etc. The `Tests` directory at the root of the repo contains all the tests, it mimics the folder structure of the main Lib/fontTools package directory. 
Let me know if you need more help

**Comment by schriftgestalt** [comment]
I expected that info in the repo. Sorry for my ignorance. Thanks a lot. This is all I need.

**Reply by anthrotype** [reply]
no problem! It's not a bad idea if the Tests directory contained a README..
